src/components/Integrations/DopeBox.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Card, CardBody, CardHeader, CardTitle, ExpandableSection, Gallery, GalleryItem, Icon, Text, TextContent, TextList, TextListItem, TextVariants } from '@patternfly/react-core';
+import { CogIcon } from '@patternfly/react-icons';
+import { LockIcon } from '@patternfly/react-icons';
+import { BellIcon } from '@patternfly/react-icons';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import '../../app/App.scss';
+
+const DopeBox = () => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
+  const onToggle = (_event: React.MouseEvent, isExpanded: boolean) => {
+    setIsExpanded(isExpanded);
+  };
+  return (
+    <Card className="noti-c-card-getting-started pf-v5-u-mb-xl">
+      <ExpandableSection className="pf-u-w-100 pf-u-flex-direction-row" toggleText={isExpanded ? 'Making use of Integrations' : 'Making use of Integrations'} onToggle={onToggle} isExpanded={isExpanded}>
```

Comment:
Since the toggle text doesn't change, it shouldn't need the condition.
```suggestion
      <ExpandableSection className="pf-u-w-100 pf-u-flex-direction-row" toggleText='Making use of Integrations' onToggle={onToggle} isExpanded={isExpanded}>
```